packages/circuit-ui/components/RadioButton/RadioButton.tsx

```diff
@@ -149,7 +164,9 @@ const inputBaseStyles = css`
   }
 
   &:disabled + label,
-  &[disabled] + label {
+  &:disabled + label span,
```

Comment:
The description should have `color: var(--cui-fg-subtle-disabled);` when disabled (note the `subtle` vs the current `normal`).